mkdocs/compat.py

```diff
@@ -0,0 +1,21 @@
+import sys
```

Comment:
... I don't have a strong preference which way we go, but we just need to be consistent. So if we really go back to compat.by there are a bunch of imports that need to be updated.